addons/website_slides_survey/models/slide_channel.py

```diff
@@ -2,9 +2,29 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
 from odoo import fields, models
+from odoo.osv import expression
 
 
 class Channel(models.Model):
     _inherit = 'slide.channel'
 
     nbr_certification = fields.Integer("Number of Certifications", compute='_compute_slides_statistics', store=True)
+
+    def _remove_membership(self, partner_ids):
+        """Unlink user_input from the slide.
```

Comment:
"unlink" sounds a bit much like we are deleting things.

Would reword to something like "Remove the relationship between the user_input and the slide".
(To apply to commit message as well).